plugins/content/transcript/lib/api/filters/KalturaTranscriptAssetFilter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaTranscriptAssetFilter extends KalturaAttachmentAssetFilter
```

Comment:
KalturaTranscriptAssetFilter must extend KalturaTranscriptAssetBaseFilter that must extend KalturaAttachmentAssetFilter.
If you were using the filters generator is would generate the right hierarchy for you.